app/backend/submissions/models.py

```diff
@@ -14,12 +14,10 @@
 from decimal import Decimal
 import uuid
 
-from django.db import models
+from django.contrib.gis.db import models
 from django.core.validators import MinValueValidator
-
 from gwells.models import AuditModel
```

Comment:
AuditModel in `app/backend/gwells/models` will need to have the version of models from `django.contrib.gis.db` too, since it's a base class for our other models.

Reply:
Agreed.. but I do see still a mix of `from django.db` and `from django.contrib.gis.db`.. mainly the auto-generated migration files.   We'll have to test to make sure that's okay..

Reply:
done **https://github.com/bcgov/gwells/blob/87b6ed7920ada6858266d68f1d74aadbc0f061e0/app/backend/wells/models.py#L15**